Mathlib/Tactic/MinImports.lean

```diff
@@ -133,6 +133,40 @@ def previousInstName : Name → Name
     .str init newTail
   | nm => nm
 
+/--
+`getDeclName cmd id` takes a `Syntax` input `cmd` and returns the `Name` of the declaration defined
+by `cmd`.
+-/
+def getDeclName (cmd : Syntax) : CommandElabM Name := do
```

Comment:
I had tried something with a different approach: [Mathlib.Linter.getNamesFrom](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Tactic/DeclarationNames.html#Mathlib.Linter.getNamesFrom).  I am not sure if it is really more usable, since there are *a lot* of internal declarations that may cause alternative problems.

Reply:
Ooh, good suggestion to use the file positions! I think the current approach works fine, since any dependency on auxiliary declarations should be picked up by traversing the term, right?